### PR TITLE
Facebook admins meta tag and Facebook options spec

### PR DESCRIPTION
**Pull request by miguelfrde**
[Domain insights](https://www.facebook.com/insights) from facebook linked to a personal account work with

```html
<meta property="fb:admins" content="{{ facebook id }}" />
```

Added a spec for all the facebook options too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,12 +60,14 @@ The SEO tag will respect any of the following if included in your site's `_confi
     username: benbalter
   ```
 
-* `facebook:app_id` (a Facebook app ID for Facebook insights), and/or `facebook:publisher` (a Facebook page URL or ID of the publishing entity). You'll want to describe one or both like so:
+* `facebook:app_id` (a Facebook app ID for Facebook insights), `facebook:publisher` (a Facebook page URL or ID of the publishing entity) and/or `facebook:admins`
```
Comment (benbalter) [inline README.md]: To improve readability, I'd break this into a short intro and three bullets like we do for `social`.

Comment (miguelfrde) [inline README.md]: Yeah that's better, updated!

**Comment by benbalter** [comment]
Thanks @miguelfrde. LGTM. But will let it sit for a bit to allow others the chance to provide feedback.

**Comment by pathawks** [comment]
Yay specs! :tada:
LGTM :+1:

**Comment by benbalter** [comment]
Thanks again @miguelfrde! 🌷 🐢 🎉

**Pull request merged**